plugins/es-discovery-ec2/pom.xml

```diff
@@ -34,9 +34,16 @@
     </dependency>
 
     <dependency>
-      <groupId>com.amazonaws</groupId>
-      <artifactId>aws-java-sdk-ec2</artifactId>
-      <version>${versions.aws}</version>
+      <groupId>software.amazon.awssdk</groupId>
+      <artifactId>ec2</artifactId>
+      <version>2.21.8</version>
+    </dependency>
+
+    <dependency>
+      <groupId>software.amazon.awssdk</groupId>
+      <artifactId>apache-client</artifactId>
+      <version>2.21.8</version>
+      <scope>compile</scope>
```

Comment:
without this the following import will cause compile error `import software.amazon.awssdk.http.apache.ApacheHttpClient;`